### DISCUSSION TITLE
Copilot completions disabled automatically.

### DISCUSSION THREAD

**Discussion by Miroxyz** [community/community#74763] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I always need to enable it manually. I'm in WebStorm 2023.2.4.

**Comment by jamirou** [comment]
You may have disabled Copylot in the settings. Check the Settings > Editor > Code Assist > Other page in WebStorm to make sure that the Enable Copylot option is checked or You may have a conflicting plugin installed. Some plugins may conflict with Copylot, preventing it from working properly. Try disabling any plugins that you think might be conflicting with Copylot and see if that resolves the issue.
There may be a bug with Copylot. If you have tried all of the above suggestions and you are still having trouble with Copylot, you may want to report a bug to JetBrains. You can do this by going to Help > Report Bug in WebStorm.
